### ISSUE TITLE
Tests use quirk files installed in system instead of local versions

### ISSUE THREAD

**Issue by marv** [fwupd/fwupd#4244]
When updating fwupd from 1.6.3 to 1.7.4 two tests failed for me with the following error:
```
>>> MALLOC_PERTURB_=195 G_TEST_BUILDDIR=/var/tmp/paludis/build/sys-apps-fwupd-1.7.4/work/_build/plugins/ata G_TEST_SRCDIR=/var/tmp/paludis/build/sys-apps-fwupd-1.7.4/work/fwupd-1.7.4/plugins/ata /var/tmp/paludis/build/sys-apps-fwupd-1.7.4/work/_build/plugins/ata/ata-self-test
―――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――― ✀  ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――
stdout:
# random seed: R02S47214c8f88549119f35d57d0fbb07206
1..2
# Start of fwupd tests
# Start of ata tests
# GLib-GIO-DEBUG: _g_io_module_get_default: Found default implementation gvfs (GDaemonVfs) for ?gio-vfs?
# XbSilo-DEBUG: attempting to load /var/tmp/paludis/build/sys-apps-fwupd-1.7.4/temp/.E38YG1
# XbSilo-DEBUG: failed to load silo: blob too small
Bail out! FuQuirks-FATAL-WARNING: [INTEL_SPI_CHIPSET\APL] Flags = PCH is invalid: P is not lowercase
stderr:

(/var/tmp/paludis/build/sys-apps-fwupd-1.7.4/work/_build/plugins/ata/ata-self-test:3250226): FuQuirks-WARNING **: 08:50:02.104: [INTEL_SPI_CHIPSET\APL] Flags = PCH is in
valid: P is not lowercase
―――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――
```

Running the test with verbose output showed that it's using the quirk file already installed in the system (`/usr/share/fwupd/quirks.d/intel-spi.quirk`)
```
# XbSilo-DEBUG: compiling /usr/share/fwupd/quirks.d/intel-spi.quirk:ctime=1633365579.446895?
Bail out! FuQuirks-FATAL-WARNING: [INTEL_SPI_CHIPSET\APL] Flags = PCH is invalid: P is not lowercase

(./plugins/nvme/nvme-self-test:64061): FuQuirks-WARNING **: 14:16:54.327: [INTEL_SPI_CHIPSET\APL] Flags = PCH is invalid: P is not lowercase
Trace/breakpoint trap (core dumped)
```

Looking at the diff of that quirk between 1.6.3 and 1.7.4 I see a bunch of:
```
-Flags = PCH
+Flags = pch
```

So I figured that's the source of the problem and that the tests should use the local, uninstalled quirk files? Is there a mechanism for that?

Best regards,
Marv

**Comment by superm1** [comment]
Maybe https://github.com/fwupd/fwupd/pull/4245 can help, can you try it?

**Comment by marv** [comment]
@superm1 Impressive, thanks for the quick fix! I can confirm that updating from 1.6.3 to current HEAD works without problems now :+1:
